mtools/mlaunch/mlaunch.py

```diff
@@ -1078,8 +1067,8 @@ def _filter_valid_arguments(self, arguments, binary="mongod", config=False):
             line = line.lstrip()
             if line.startswith('-'):
                 argument = line.split()[0]
-                # exception: don't allow --oplogSize for config servers
-                if config and argument == '--oplogSize':
+                # exception: don't allow --oplogSize, --storageEngine, --smallfiles, and --nojournal for config servers
```

Comment:
The comment doesn't need to repeat all the same options in the code below .. it should clarify the intent, eg: "Ignore arguments that should not be passed to config servers".